src/fides/api/api/v1/endpoints/dataset_endpoints.py

```diff
@@ -214,7 +214,8 @@ def put_dataset_configs(
         db.query(DatasetConfig).filter(
             DatasetConfig.connection_config_id == connection_config.id,
             DatasetConfig.fides_key.in_(config_keys_to_remove),
-        ).delete()
+        ).delete(synchronize_session=False)
```

Comment:
what's the default `synchronize_session` if not provided?

Reply:
The default is `evaluate`. We're using `False` here since we don't need to maintain the deleted objects in the session state, we're just removing records that match our filter criteria.